docs/gwcs/points_to_wcs.rst

```diff
@@ -0,0 +1,51 @@
+.. _user_api:
+
+Fitting a WCS to input pixels & sky positions
+=============================================
+
+Suppose we have an image where we have centroid positions for a handful of sources, and we have matched these 
+positions to an external catalog to obtain (RA, Dec). If this data is missing or has inaccurate WCS information,
+it is useful to fit or re-fit a GWCS object with this matched list of coordinate pairs to be able to transform
+between pixel and sky. 
+
+This example shows how to use the wcs_from_point tool to fit a WCS to a matched set of 
```

Comment:
Use `~gwcs.wcstoolf.wcs_from_point` (in single backticks) in order to link to the function.